crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/account_deposit.rs

```diff
@@ -0,0 +1,107 @@
+use crate::prelude::*;
+use std::collections::HashMap;
+
+/// Represents an account deposit, which includes specified and unspecified resources.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
+pub struct AccountDeposit {
+    specified_resources: HashMap<ResourceAddress, SimpleResourceBounds>,
+    unspecified_resources: UnspecifiedResources,
+}
+
+impl AccountDeposit {
+    pub fn new(
+        specified_resources: HashMap<ResourceAddress, SimpleResourceBounds>,
+        unspecified_resources: UnspecifiedResources,
+    ) -> Self {
+        Self {
+            specified_resources,
+            unspecified_resources,
+        }
+    }
+}
+
+impl From<(ScryptoAccountDeposit, NetworkID)> for AccountDeposit {
+    fn from(value: (ScryptoAccountDeposit, NetworkID)) -> Self {
+        let (scrypto_value, network_id) = value;
+        let specified_resources = scrypto_value
+            .specified_resources()
+            .into_iter()
+            .map(|(address, bounds)| {
+                ((address.clone(), network_id).into(), bounds.clone().into())
```

Comment:
Prefer using `TheTypeWeWant::from(unwanted)` instead of `unwanted.into()`. I've used `into()` A LOT in the past, but changed my mind, since reading the code is so much easier - especially in code review in Github were we get no hint from an IDE, if we can see the type we are converting into explicitly.

have you run `cargo clippy --fix --allow-staged` btw? I think the `address.clone()` might not be needed? Not if it is `Copy`, but perhaps it is not `Copy`.